src/Header/Header.scss

```diff
@@ -59,10 +59,15 @@ header {
 
   @mixin main-menu-list {
     list-style-type: none;
-    margin-right: -2rem;
```

Comment:
This stuff just fixes some responsiveness issues that appeared when we added "Services & Pricing" to the main menu